volumes/short_term_counting_program/sql/create-table-fhwa_classes.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE traffic.fhwa_classes (
+    class_id int,
+    class_desc text
+);
+
+INSERT INTO traffic.fhwa_classes VALUES
+(1, 'motorcycles'),
+(2, 'passenger cars'),
+(3, 'two axle, 4 tire single units'),
+(4, 'buses'),
+(5, 'two axis, 6 tire single units'),
+(6, 'three axle single units'),
+(7, 'four or more axle single units'),
+(8, 'four or less axle single trailers'),
+(9, 'five axle single trailers'),
+(10, 'six or more axle single trailers'),
+(11, 'five or less axle multi-trailers'),
+(12, 'six axle multi-trailers'),
+(13, 'seven or more axle multi-trailers');
+
+ALTER TABLE traffic.fhwa_classes OWNER TO traffic_admins;
+GRANT SELECT ON TABLE traffic.fhwa_classes TO bdit_humans;
+
+COMMENT ON TABLE traffic.fhwa_classes IS E''
```

Comment:
What's up with this E? Does it do something neat?

Reply:
I found this method [here](https://stackoverflow.com/a/75490356).
I needed it in order to include the `noqa` (for lines that are too long) in the middle of a multi line comment!

Reply:
Approved!